Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 ruby '2.2.0'
 
 gem 'rails', '4.2.0'
+gem 'json'
```

Comment:
Oh, whoops! It's because I have to add `gem 'json', github: 'hsbt/json', ref: 'ruby220'` to my Gemfile on Ubuntu :(